x-pack/plugins/fleet/common/constants/output.ts

```diff
@@ -13,6 +13,7 @@ export const outputType = {
   Elasticsearch: 'elasticsearch',
   Logstash: 'logstash',
   Kafka: 'kafka',
+  RemoteElasticsearch: 'remote-elasticsearch',
```

Comment:
nit: I think we use `remote_elasticsearch` will be more consistant with the other fleet server/beats config, but not sure we have clear guidelines for that

Reply:
done